.github/pull_request_template.md

```diff
@@ -0,0 +1,6 @@
+### Pull Request checklist ###
+<!-- Before submitting the PR, please address each item -->
+- [ ] **Quality**: This PR builds and `cargo test` runs cleanly
+- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
+- [ ] **Changelog**: This PR includes a changelog entry or an explanation of why it does not need one
```

Comment:
I think new contributors may be confused by this, since they'll almost certainly not be touching anything that will effect our semver story. We don't follow semver for our rust crates (they're all kinda perma-0.1.0), since they aren't on crates.io (except for ffi_support).

Reply:
To clarify, are you saying that contributors probably won't need to add changelog entries at all, or that they won't need to worry about semver specifically?

(Partly I added this changelog item to remind us of it ourselves, FWIW).